app/recordtransfer/templates/recordtransfer/banner.html

```diff
@@ -1,6 +1,20 @@
-<div class="banner-container">
-    <div>
+{% load i18n %}
+<div class="banner-container flex items-center justify-between gap-6 w-full">
+    <div class="flex items-center">
```

Comment:
This phone line gets squished when the screen is narrow. Let's hide it on `sm` screens. I was worried about doing this but if you look at https://nctr.ca they do the same thing so I'm OK with doing that here.

```suggestion
    <div class="hidden sm:flex items-center">
```